src/app/payment/paymentMethod/PaypalCommercePaymentMethod.tsx

```diff
@@ -29,6 +29,9 @@ const PaypalCommercePaymentMethod: FunctionComponent<PaypalCommercePaymentMethod
                 setSubmitted(true);
                 submitForm();
             },
+            onError: (error: any) => {
```

Comment:
You don't need `any` as the parameter should already be typed.

Reply:
fixed